lib/Kind.py

```diff
@@ -1,38 +1,43 @@
 import common
 import time
 import os
+from robot.api import logger
```

Comment:
why changing to logger api instead of continuing to use `print` statements. Our focus here is on something else, let's do not too many things at once.

Reply:
`print` statement does not print logs to robot reporting tool, whereas `logger` by `robot.api` does. It is easier to toggle reporting mode with `debug/info` switch in report.html

Reply:
sure, but let's tackle this in another PR - this PR should be about running tests against an arbitrary cluster. Let's try to do one thing at once. Improving logging should be done in a different PR.